index.html

```diff
@@ -435,22 +435,32 @@
             <a>Queue a task</a> on the <a>application life-cycle task
             source</a> to do the following:
             <ol>
-              <li>Let <var>event</var> be a <a data-lt=
-              "construct a BeforeInstallPromptEvent">newly constructed</a> <a>
-                BeforeInstallPromptEvent</a> named
-                "<a>beforeinstallprompt</a>", which is cancelable.
+              <li>Let <var>event</var> be a newly constructed
+              <a>BeforeInstallPromptEvent</a> named
+              <code>beforeinstallprompt</code>, with its <code>cancelable</code>
+              attribute initialized to true.
               </li>
-              <li>
-                <a>Fire</a> <var>event</var> at the <a>Window</a> object of the
-                <a>top-level browsing context</a>.
+              <li>Let <var>showPrompt</var> be the result of <a>firing</a>
+              <var>event</var> at the <a>Window</a> object of the <a>top-level
+              browsing context</a>.
               </li>
-              <li>If <var>event</var>'s <a>canceled flag</a> is not set, then,
-              <a>in parallel</a>, <a>request to present an install prompt</a>
-              with <var>event</var>.
+              <li>If <var>showPrompt</var> is true, then, <a>in parallel</a>,
+              <a>request to present an install prompt</a> with
+              <var>event</var>.
               </li>
             </ol>
           </li>
         </ol>
+        <div class="issue">
+          Implementations may wish to show a prompt if and only if the site
```

Comment:
I think comma around "and only if" is correct.

(Why can't I do many comments together? Maybe this pull request was started so long ago that it's grandfathered out.)